HospitalSystem.ConsoleUI/Director/Surveys/DoctorSurveyUI.cs

```diff
@@ -0,0 +1,130 @@
+using HospitalSystem.Core;
+using HospitalSystem.Core.Surveys;
+
+namespace HospitalSystem.ConsoleUI.Director.Surveys;
+
+public class DoctorSurveyUI : SurveyUI
+{
+    private List<DoctorSurvey> _loadedSurveys;
+
+    public DoctorSurveyUI(Hospital hospital) : base(hospital)
+    {
+        _loadedSurveys = new();
+    }
+
+    private void RefreshSurveys()
+    {
+        _loadedSurveys = _hospital.SurveyService.GetAllDoctor().ToList();
```

Comment:
Should be called GetAllDoctors? plural not singular

Reply:
This does not get Doctors. It gets DoctorSurveys

Reply:
Okay!